process/project-lifecycle.md

```diff
@@ -1,156 +1,168 @@
 
-Projects that wish to move from Incubating to Graduation should open a PR confirming the following criteria:
-* Have maintainers from at least two organizations.
-* Have achieved and maintained an [OpenSSF Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
-* Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation. The TAC, or another WG, may provide assistance in acquiring such an audit.
-* Explicitly define project governance and process to be granted maintainership within the project team. This is preferably laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus maintainers.
-* Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website). For a specification, have a list of adopters for the implementation(s) of the spec.
-* Please include a short one-page narrative based off the Graduation template, no more than 500 words.
 
 #### Submission Forms for Incubating
 
 1. [Proposal](TBD)
 2. [Legal](TBD)
 
-#### Submission Forms for Incubating to Graduation
-
-1. [Proposal](TBD)
-2. [Legal](TBD)
 
 ### Graduated
 
-Graduated projects signal the highest level of maturity for an OpenSSF project.
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
+
+#### Project Reponsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices
+* Implement, practice, and refine mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+
+#### Project Support
+* Receives guidance on technical direction from TAC 
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
```

Comment:
I think this one will take some negotiation with the GB - they currently control the entire blog process and are likely to continue to want some oversight/direction over the content.

Reply:
💯 This may be something that can be generally managed with support from OpenSSF staff.  Regardless we should be clear on what's those areas of responsibility and hand off look like.

Reply:
Is this a question we can delegate to OpenSSF Marketing Committee? ( cc @jorydotcom) Sounds like the question something like, "What is a blog process that gives access to project maintainers but gives the GB editorial oversight and the TAC awareness?"